examples/language/lib/patterns/switch.dart

```diff
@@ -26,10 +26,10 @@ void main() {
   }
   // #enddocregion constant-pattern
 
+  // #docregion list-pattern
   const a = 'a';
   const b = 'b';
   var obj = Object();
```

Comment:
Thanks for doing this! Could you move the `obj` declaration line outside of the `docregion`? That part is pretty arbitrary.

```suggestion
  var obj = Object();
  // #docregion list-pattern
  const a = 'a';
  const b = 'b';
```

Reply:
Sounds good. Would you also like me to remove it from the 'src/language/patterns.md' file too?

Reply:
Thanks! Yes please :)

The markdown snippet is matched against the `.dart` file to make sure they match.

Reply:
done. :)